packages/harmony/src/components/avatar/Avatar.stories.tsx

```diff
@@ -0,0 +1,60 @@
+import type { Meta, StoryObj } from '@storybook/react'
+
+import { Box } from 'components/layout/Box'
+import { Flex } from 'components/layout/Flex'
+import { Paper } from 'components/layout/Paper'
+// TODO: Get final image assets from Sammie
+import shadowBackground from 'storybook/assets/shadowBackground.jpeg'
+
+import { Avatar } from './Avatar'
+
+const meta: Meta<typeof Avatar> = {
+  title: 'Components/Avatar',
+  component: Avatar,
+  parameters: {
+    controls: {}
+  },
+  render: (props) => {
+    return (
+      <Box w={80} h={80}>
+        <Avatar {...props} />
+      </Box>
+    )
+  }
+}
+
+export default meta
+
+type Story = StoryObj<typeof Avatar>
+
+export const Default: Story = {
+  args: {
+    url: shadowBackground
+  }
+}
+
+export const Strong: Story = {
+  args: {
+    variant: 'strong',
+    url: shadowBackground
+  },
+  render: (props) => (
+    <Paper w={350} h={160}>
+      <Box
+        w={80}
+        h={80}
+        style={{ position: 'absolute', top: '40px', left: '32px' }}
+      >
+        <Avatar {...props} />
```

Comment:
Per the don't in Avatar docs, we don't want people messing with the avatar's styles. This is just needed because the default 'auto' size on Avatar will be 100% of 0px height and width if there's no box to put it in. In real use cases, you'd have a box to put it in or use the explicit sizings. Alternately, we could make the 'large' size the default, but that was pretty small on storybook